docs/3.api/2.composables/use-preview-mode.md

```diff
@@ -5,12 +5,63 @@ description: "Use usePreviewMode to check and control preview mode in Nuxt"
 
 # `usePreviewMode`
 
+Preview Mode allows you to see how your changes would be displayed on a live site without revealing them to users.
+
 You can use the built-in `usePreviewMode` composable to access and control preview state in Nuxt. If the composable detects preview mode it will automatically force any updates necessary for [`useAsyncData`](/docs/api/composables/use-async-data) and [`useFetch`](/docs/api/composables/use-fetch) to rerender preview content.
 
 ```js
 const { enabled, state } = usePreviewMode()
 ```
 
+## Example
+
+The example below creates a page where part of a content is rendered only in `Preview mode`.
+
+```vue [pages/some-page.vue]
+<script setup>
+const route = useRoute()
+const { enabled, state } = usePreviewMode({
+  shouldEnable: () => {
+    return route.query.customPreview === 'true'
+  },
```

Comment:
maybe it would be worth leaving the example below as an example of `shouldEnable` and modify this one so it shows how it works out of the box, without `shouldEnable`?

Reply:
I've been thinking about this while making my PR and the idea was to show in the example as much as possible. But I understand that might be overwhelming. I will update the example. Thank you

Reply:
I think it's not necessary to have whole example at the bottom, because there already is an example  for `shouldEnable` :

```
export function useMyPreviewMode () {
  return usePreviewMode({
    shouldEnable: () => {
      return !!route.query.customPreview
    }
  });
}
```

Reply:
sounds good